File: Chatmvp - Java/app/src/main/java/nosql/anew/chatmvp/views/IChatView.java
package nosql.anew.chatmvp.views;

import java.util.ArrayList;

import nosql.anew.chatmvp.models.ChatPojo;

/**
 * Created by saksham on 26/6/17.
 */

public interface IChatView {
    void updateList(ArrayList<ChatPojo> list);
    void clearEditText();
}

File: Chatmvp - Kotlin/app/src/main/java/nosql/anew/kotlinmvp/views/IChatView.kt
package nosql.anew.kotlinmvp.views

import nosql.anew.kotlinmvp.models.ChatPojo
import java.util.ArrayList

/**
 * Created by Sam on 27/04/18.
 */
interface IChatView {
    fun updateList(list: ArrayList<ChatPojo>)
    fun clearEditText()
}